content/en/observability_pipelines/setup/datadog.md

```diff
@@ -215,13 +238,16 @@ There are no provider-specific requirements for RPM-based Linux.
     sudo systemctl restart observability-pipelines-worker
     ```
 
-[1]: /resources/yaml/observability_pipelines/quickstart/linux.yaml
+[1]: /resources/yaml/observability_pipelines/quickstart/datadog.yaml
 {{% /tab %}}
 {{< /tabs >}}
 
 ### Load balancing
 
 {{< tabs >}}
+{{% tab "Docker" %}}
+No built-in support for load-balancing is provided, given the single-machine nature of the installation. You will need to provision your own load balancers using whatever your company's standard is.
```

Comment:
I'm not 100% sold on the idea of leaving load balancing instructions here, since I'd rather leave a note that Docker used like this is not a supported production configuration. If they're testing on their Mac, they shouldn't need this.

Reply:
How about something like this?
> Production-oriented setup is not covered by these Docker instructions, and you should defer to your company's standards for load balancing in containerized environments. If you are testing on your local machine, configuring a load balancer is unnecessary.

Reply:
I like that!